chef/cookbooks/hyperv/recipes/openstack_install_networking-hyperv.rb

```diff
@@ -0,0 +1,26 @@
+raise if not node[:platform_family] == "windows"
+
+cookbook_file "#{node[:cache_location]}#{node[:openstack][:networking_hyperv][:file]}" do
+	source node[:openstack][:networking_hyperv][:file]
```

Comment:
Style/IndentationWidth: Use 2 (not 1) spaces for indentation. (https://github.com/bbatsov/ruby-style-guide#spaces-indentation)<br>Style/Tab: Tab detected. (https://github.com/bbatsov/ruby-style-guide#spaces-indentation)